client/components/game-list.jsx

```diff
@@ -12,10 +10,37 @@ export default function GameList(props) {
               key={single.name}
               title={single.name}
               img={single.image.super_url}
+              id={single.id}
+              description={single.description}
+              deck={single.deck}
+              getGame={props.getGame}
             />
           );
         })
       }
     </ul>
   );
 }
+
+function SingleGame(props) {
+  const { id, title, img, deck, description } = props;
+  const bringBack = () => {
+    props.getGame(id, title, img, deck, description);
+  };
+  return (
```

Comment:
You can get rid of this. Put the whole thing in an `<a>` with `display: block`. Let the hash change take over.